priam/src/main/java/com/netflix/priam/cluster/management/SchemaConstant.java

```diff
@@ -0,0 +1,25 @@
+package com.netflix.priam.cluster.management;
+
+/**
+ * Created by aagrawal on 3/6/18.
+ */
+public class SchemaConstant {
+    public static final String SYSTEM_KEYSPACE_NAME = "system";
```

Comment:
If these constants are used only once, do you need these constants? also, instead of having these many variables, can you try any regex? or more common utility method `isSystemKeysapce` so that you can use it across the board in Priam if you need it in future?